github/orgs_members.go

```diff
@@ -270,3 +270,29 @@ func (s *OrganizationsService) RemoveOrgMembership(user, org string) (*Response,
 
 	return s.client.Do(req, nil)
 }
+
+// ListPendingOrgInvitations returns a list of pending invitations.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
+func (s *OrganizationsService) ListPendingOrgInvitations(org int, opt *ListOptions) ([]*Invitation, *Response, error) {
```

Comment:
Hmm. Looking at the methods of `OrganizationsService` more closely, it seems there's many, many uses of `Org`. So I'm afraid it might make sense to stay consistent with that and keep this as is.

![image](https://cloud.githubusercontent.com/assets/1924134/22454798/81d22e8e-e756-11e6-85a7-3bf1c94c0b26.png)

It seems valuable to keep Org because it makes it more clear that the pending invitation applies to org, rather than team (which also exists).